ksqldb-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/CreateSourceCommand.java

```diff
@@ -88,9 +88,6 @@ public Boolean isOrReplace() {
   }
 
   private static void validate(final LogicalSchema schema, final boolean windowed) {
-    if (schema.valueContainsAny(SystemColumns.systemColumnNames())) {
```

Comment:
It's a bit confusing that this change is lumped into this PR. I thought the intention was to limit this PR to non-functional changes (adding new code that will be called in a subsequent PR), but this is changing existing production code. It'd be nice to move this into a separate PR so it's easier to reason about test coverage.

Reply:
Since systemColumnNames was being called with it's current version and the current version was 1, this was causing issues and I wrongly anticipated we'd need to remove validation at this stage to allow QTTs to work. Fixed and removed these changes.